Movecraft/build.gradle.kts

```diff
@@ -36,6 +45,23 @@ tasks.processResources {
         expand(mapOf("projectVersion" to project.version))
     }
 }
+tasks {
+    supportedMinecraftVersions.forEach { serverVersion ->
+        register<RunServer>("run-$serverVersion") {
+            minecraftVersion(serverVersion)
+            jvmArgs("-DPaper.IgnoreJavaVersion=true", "-Dcom.mojang.eula.agree=true")
+            group = "run paper"
+            runDirectory.set(file("run-$serverVersion"))
+            downloadPlugins {
+                url("https://dev.bukkit.org/projects/worldedit/files/5564367/download")
```

Comment:
Worldedit is available from Hangar: https://hangar.papermc.io/EngineHub/WorldEdit

Reply:
oh thanks for some reason I couldn't find it